Buildings/Controls/OBC/ASHRAE/PrimarySystem/ChillerPlant/Economizer/Subsequences/Validation/Tuning_uEcoSta_uTowFanSpeMax.mo

```diff
@@ -0,0 +1,141 @@
+    annotation (Placement(transformation(extent={{20,-20},{40,0}})));
+
+  Buildings.Controls.OBC.CDL.Continuous.Sources.Constant maxTowFanSpeSig3(
+    final k=1)
+    "Maximum cooling tower fan speed signal"
+    annotation (Placement(transformation(extent={{20,-70},{40,-50}})));
+
+equation
+  connect(ecoSta.y,wseTun.uWseSta)  annotation (Line(points={{-99,110},{-70,110},
+          {-70,75},{-42,75}},
+                            color={255,0,255}));
+  connect(maxTowFanSig1.y, min1.u1) annotation (Line(points={{41,70},{50,70},{50,
+          56},{58,56}}, color={0,0,127}));
+  connect(cooTowFanSta1.y, min1.u2) annotation (Line(points={{41,28},{48,28},{48,
+          44},{58,44}}, color={0,0,127}));
+  connect(min1.y, wseTun1.uTowFanSpeMax) annotation (Line(points={{81,50},{90,
+          50},{90,65},{98,65}}, color={0,0,127}));
+  connect(ecoSta1.y,wseTun1.uWseSta)  annotation (Line(points={{41,110},{70,110},
+          {70,75},{98,75}}, color={255,0,255}));
+  connect(ecoSta2.y,wseTun2.uWseSta)  annotation (Line(points={{-99,10},{-70,10},
+          {-70,-25},{-42,-25}}, color={255,0,255}));
+  connect(maxTowFanSpeSig2.y, wseTun2.uTowFanSpeMax) annotation (Line(points={{
+          -99,-50},{-70,-50},{-70,-35},{-42,-35}}, color={0,0,127}));
+  connect(maxTowFanSpeSig0.y, wseTun.uTowFanSpeMax) annotation (Line(points={{-99,
+          50},{-70,50},{-70,65},{-42,65}}, color={0,0,127}));
+  connect(ecoSta3.y,wseTun3.uWseSta)  annotation (Line(points={{41,-10},{70,-10},
+          {70,-45},{98,-45}}, color={255,0,255}));
+  connect(wseTun3.uTowFanSpeMax, maxTowFanSpeSig3.y) annotation (Line(points={{
+          98,-55},{70,-55},{70,-60},{41,-60}}, color={0,0,127}));
+annotation (
+ experiment(StopTime=28800.0, Tolerance=1e-06),
+  __Dymola_Commands(file="modelica://Buildings/Resources/Scripts/Dymola/Controls/OBC/ASHRAE/PrimarySystem/ChillerPlant/Economizer/Subsequences/Validation/Tuning_uEcoSta_uTowFanSpeMax.mos"
+    "Simulate and plot"),
+  Documentation(info="<html>
+<p>
+This example validates
+<a href=\"modelica://Buildings.Controls.OBC.ASHRAE.PrimarySystem.ChillerPlant.Economizer.Subsequences.Tuning\">
+Buildings.Controls.OBC.ASHRAE.PrimarySystem.ChillerPlant.Economizer.Subsequences.Tuning</a>.
+</p>
+</html>", revisions="<html>
+<ul>
+<li>
+October 15, 2018, by Milica Grahovac:<br/>
+First implementation.
+</li>
+</ul>
+</html>"),
+Icon(graphics={
+        Ellipse(lineColor = {75,138,73},
+                fillColor={255,255,255},
+                fillPattern = FillPattern.Solid,
+                extent = {{-100,-100},{100,100}}),
+        Polygon(lineColor = {0,0,255},
+                fillColor = {75,138,73},
+                pattern = LinePattern.None,
+                fillPattern = FillPattern.Solid,
+                points = {{-36,60},{64,0},{-36,-60},{-36,60}})}),Diagram(
+        coordinateSystem(preserveAspectRatio=false, extent={{-140,-120},{140,140}}),
+        graphics={
+        Text(
```

Comment:
Would be better to make the text showing in larger font size, by changing the `extent={{-106,-80},{-46,-102}}` to say, `extent={{-124,-80},{-18,-92}}`?